WordPress/src/main/res/layout/reader_related_post.xml

```diff
@@ -1,51 +1,109 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    individual related post shown in ReaderRelatedPostsView
+ -->
+
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="?android:selectableItemBackground"
-    android:padding="@dimen/reader_related_post_margin">
+    android:layout_marginBottom="@dimen/margin_extra_large"
+    android:background="?android:selectableItemBackground">
+
+    <org.wordpress.android.widgets.WPNetworkImageView
+        android:id="@+id/image_featured"
+        android:layout_width="@dimen/reader_related_post_image_width"
+        android:layout_height="match_parent"
+        android:layout_marginRight="@dimen/margin_large"
+        android:scaleType="fitXY"
+        android:visibility="gone"
+        tools:src="@drawable/box_with_pages_top"
+        tools:visibility="visible" />
 
-    <LinearLayout
+    <RelativeLayout
+        android:id="@+id/layout_related_post_site_header"
```

Comment:
This is more like a comment just to put it out there: It seems that the header click acts as a click on the related post itself and opens the post. It felt counter-intuitive the first couple of times I tried it. I was expecting it to go to the site (in Reader) instead. Apart from the smaller size overall, the header looks like the relevant headers elsewhere, and there the click opens the site.

This is not a blocking comment though.